src/api.js

```diff
@@ -221,6 +238,23 @@ const getTopology = function(res, args) {
     }
 }
 
+const getRecord = function(res, args) {
+    let result  = [];
+    let records = data.GetRecords();
+    let ids     = Array.isArray(args.id) ? args.id : [args.id];
+
+    ids.forEach(id => {
```

Comment:
generally in js we avoid to mutate structures (so whenever possible we create const instead let
`const result = ids.map(id => records[id] || null).filter(Boolean) ` 

filter(Boolean) removes null

In case we don't want to use .filter(Boolean)
```
const result = ids.reduce((acc,id) => {
   if(records[id]) {
      acc.push(records[id])
    }

    return acc
},[])
```